metrics/jenkins/nightly-build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export KUBE_SKIP_UPDATE="y"
+export KUBE_PROMPT_FOR_UPDATE="N"
+
+: ${ENV_SCRIPT_DIR:?"Need to set ENV_SCRIPT_DIR"}
+: ${PUBLISHER_DIR:?"Need to set PUBLISHER_DIR"}
+# Assumes that most of the time we run nightly builds in k8s repo
+K8S_DIR=${K8S_DIR:-`pwd`}
+KUBEMARK_LOG_DIR=${KUBEMARK_LOG_DIR:-"/var/log"}
+
+cleanup() {
+  (source "${ENV_SCRIPT_DIR}/kubemark-env.sh" && ${K8S_DIR}/test/kubemark/stop-kubemark.sh) || true
+  (source "${ENV_SCRIPT_DIR}/k8s-cluster-env.sh" && ${K8S_DIR}/cluster/kube-down.sh) || true
+}
+
+trap cleanup EXIT
+
+source "${ENV_SCRIPT_DIR}/k8s-cluster-env.sh" && ${K8S_DIR}/cluster/kube-up.sh
+
+source "${ENV_SCRIPT_DIR}/kubemark-env.sh" && ${K8S_DIR}/test/kubemark/start-kubemark.sh
+# Even if test failed, the log file is still useful to us
```

Comment:
If the test failed, can we add a prefix to the log file like kubemark-log-failed.txt?